profiles/car.lua

```diff
@@ -194,6 +194,7 @@ properties.max_speed_for_map_matching      = 180/3.6 -- 180kmph -> m/s
 properties.use_turn_restrictions           = true
 properties.continue_straight_at_waypoint   = true
 properties.left_hand_driving               = false
+properties.version                         = 0
```

Comment:
My thinking here is that we actually don't need the profile version information outside of the scripting context.